platform/search/bundle.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Search",
+    "description": "Allows the user to search through the file tree.",
+    "extensions": {
+        "constants": [
+            {
+                "key": "GENERIC_SEARCH_ROOTS",
```

Comment:
@VWoeltjen @larkin  I just noticed that doing this method now makes 'the root object' available to be searched. Do we want that?